### ISSUE TITLE
Please disable default image sharpening in the VIPS backend

### ISSUE THREAD

**Issue by pyridine** [janko/image_processing#113]
Hi, this is related to https://github.com/carrierwaveuploader/carrierwave/issues/2481 and https://github.com/janko/image_processing/pull/67
Default image sharpening was disabled a while back in the ImageMagick backend, but not in the VIPS backend. I think it should be disabled by default in both. The default sharpening behavior is not expected and can degrade/change the image quality.

Here are the results of resizing an image in VIPS vs ImageMagick:

**_Base image_**
![base image](https://github.com/janko/image_processing/assets/10067934/5ffa635d-c7b5-47a0-9c00-af0ebca5bf7c)
**_Converted with ImageMagick backend_**
![converted with imagemagic backend, it appears the same](https://github.com/janko/image_processing/assets/10067934/950ac4c3-00d8-403a-a02d-d3f99daa7819)
**_Converted with VIPS backend_**
![converted with vips backend, it looks worse](https://github.com/janko/image_processing/assets/10067934/d7701a7b-0963-498c-893a-032aca0a113b)

As with the original issue, the VIPS-processed image appears lower quality, or at least different, which is not expected.

I used this code:
```ruby
processed = ImageProcessing::Vips # or MiniMagick
  .source(base_image)
  .resize_to_fit(1280,720) # the image is already this size, so this shouldn't change anything
  .convert("png")
  .call
```

**Comment by midnight-wonderer** [comment]
Your eyes are really good; I couldn't tell the difference.
For those who are like me, here is where to look:
![diff](https://i.imgur.com/Bzq3kTY.jpeg)
Created with: _ttps://www.diffchecker.com/image-compare/

P.S. I could see it in the original post once pointed out.

**Comment by janko** [comment]
I plan to disable automatic sharpening in libvips for the upcoming major release. I agree it's unexpected, and it would be consistent with disabling it for the MiniMagick backend. I will still keep a default sharpen mask that can be turned on.

**Comment by formigarafa** [comment]
I'm impressed with what you guys can see. Thank you for that, but I didn't. ❤️

**Comment by cmitz** [comment]
So I've encountered this with as well, but the bug is absolutely the weirdest... It seems Apple HEIC photo's (which I want to support) result in super weird artefacts when processed. This doesn't happen when I add `sharpen: false`. 🤷‍♂️ 

<img width="1920" height="1440" alt="Image" src="https://github.com/user-attachments/assets/fba4031b-dda7-4814-b578-3cff28624aad" />

AI was mumbling something about Apples ICC profile being close to `uchar` boundaries, resulting in clipping in certain areas. So now I have to do `sharpen: false` everywhere.

Long story short, can we disable sharpening by default?

**Comment by kleisauke** [comment]
@cmitz That appears to be the same issue reported in https://github.com/libvips/libvips/issues/4955, which was resolved in libvips 8.18.2.

**Comment by cmitz** [comment]
@kleisauke Wauw, good catch! I did not even look further than this issue 🫣 
Can confirm updating libvips solved the issue

**Comment by janko** [comment]
Even though newer libvips versions resolved some issues with sharpening, I will still disable it by default to prioritize correctness by default, like I did with ImageMagick.
